app/assess/templates/macros/sub_criteria_navbar.html

```diff
@@ -2,7 +2,8 @@
 
 <nav class="assessment-navigation">
     <ul class="govuk-list">
-        {%for theme in sub_criteria.themes%}
+        
+        {%for theme in sub_criteria.themes|remove_theme_name_score%}
```

Comment:
This seems like a weird way to do business logic, is there a way we can pass into the template instead?

Reply:
Will add TODO to it as it may require quite few changes and few tests would fail so it was the easiest fix. But we will look into it later